src/components/info/templates/default-template.vue

```diff
@@ -0,0 +1,152 @@
+<script setup lang="ts">
+import { defineProps } from 'vue'
+import {
+  FeatureInfoJSON,
+  FeatureJSON,
+} from '../../../services/info/feature-info.model'
+import { useTranslation } from 'i18next-vue'
+import {
+  hasAttributes,
+  isEmpty,
+  isLink,
+  showAttributesByLang,
+  hasValidFID,
+  prefixKeys,
+  getTrustedUrl,
+} from './template-utilities'
+import i18next from 'i18next'
+import ProfileFeatureInfo from '@/components/info/profile-feature-info.vue'
+
+defineProps({
+  layers: {
+    type: Object as () => FeatureInfoJSON,
+    required: true,
+  },
+})
+defineEmits<{
+  (e: 'export', payload: { feature: FeatureJSON; format: 'kml' | 'gpx' }): void
+}>()
+const { t } = useTranslation()
+const currentUrl = window.location.href
+</script>
+<template>
+  <div class="flex flex-col">
+    <div>
+      <h1 class="lux-poi-title" data-cy="defaultTemplateTitle">
+        {{ t(`${layers.layerLabel}`) }}
```

Comment:
```suggestion
        {{ t(layers.layerLabel) }}
```
unless label can be numbers?

Reply:
Ah right, I can even get rid of all the other string interpolations I think.